app/repositories/repository_registry.rb

```diff
@@ -1,13 +1,12 @@
 require "specialist_document_repository"
 require "manual_repository"
-require "specialist_document_edition"
+require "section_edition"
```

Comment:
Is this require statement necessary now that the class name matches the filename?